website/src/controllers/api-v3/groups.js

```diff
@@ -303,8 +305,8 @@ api.leaveGroup = {
       }
     }
 
-    await group.leave(user, req.query.keep);
-    res.respond(200, {});
+    let leave = await group.leave(user, req.query.keep);
+    res.respond(200, leave);
```

Comment:
Let's just return an empty object for now. That way we can add stuff later if it becomes useful.